review_practice/api-server/resolvers/room.js

```diff
@@ -0,0 +1,45 @@
+import { AuthenticationError, UserInputError } from 'apollo-server';
+
+// Validator
+const isAdultValueLessEqualThan0 = searchOptions =>
+    searchOptions && searchOptions.personnel && searchOptions.personnel.adult <= 0;
```

Comment:
```suggestion
    searchOptions?.personnel?.adult <= 0;
```
이렇게 옵셔널 체이닝을 사용하면 어떨까요?